yb-voyager/cmd/cutoverInitiate.go

```diff
@@ -27,6 +27,7 @@ var cutoverInitiateCmd = &cobra.Command{
 
 	Run: func(cmd *cobra.Command, args []string) {
 		validateExportDirFlag()
+		initMetaDB()
```

Comment:
See if we can call this function from the `rootCmd.PersistentPreRun()`. That way, it will be called for each command. I am not sure how would it pan out if the "project" itself is not created yet.

Reply:
yeah that was my concern also, thats why i didn't add it in the rootCmd itself